libs/core/testing/include/hpx/testing/performance.hpp

```diff
@@ -18,5 +19,8 @@ namespace hpx::util {
         std::string const& exec, std::size_t const steps,
         hpx::function<void()>&& test);
 
-    HPX_CORE_EXPORT void perftests_print_times();
+    // templ is a mustache-style template for the output
+    // used by the nanobench reporter
+    HPX_CORE_EXPORT void perftests_print_times(
+        std::ostream& strm = std::cout, char const* templ = nullptr);
```

Comment:
Could we keep `std::cout` out of the API?

Reply:
Sure thing! Which of the following do you think is the better way to do it?

1) Leave `std::cout` as the default stream, but handle it on the `.cpp` file with polymorphism instead of the header. 
2) Do not provide a default stream.

The original API does not accept a stream parameter

Reply:
Let's add another overload that allows to specify the ostream and handle the default case in the source file.